# HG changeset patch
# User Philip Nienhuis <prnienhuis@users.sf.net>
# Date 1458328935 -3600
#      Fri Mar 18 20:22:15 2016 +0100
# Node ID fe56e066825df8d4cdb71b5d9233cce52c8f945e
# Parent  c64e57319f081803d4510a09478e853674e90625
NEWS: announce improved textscan function

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -63,16 +63,19 @@ Summary of important user-visible change
 
  ** The quadl function now uses an absolute, rather than relative,
     tolerance for Matlab compatibility.  The default tolerance is 1e-6
     which may result in lower precision results than previous versions
     of Octave which used eps as the relative tolerance.  The quadl
     function has also been extended to return a second output with the
     total number of function evaluations.
 
+ ** The textscan function is now built-in and is much faster and much
+    more Matlab-compatible than the previous m-file version.
+
  ** Other new functions added in 4.2:
 
       deg2rad
       evalc
       hash
       im2double
       psi
       odeset
